packages/abi/src/matchers/sway-type-matchers.ts

```diff
@@ -22,7 +23,7 @@ export type SwayType =
   | 'assetId'
   | 'evmAddress'
   | 'rawUntypedPtr' // might not need it
```

Comment:
Why is this not needed again?

Reply:
I didn't have a need for it in typegen, but doing a search in our codebase for "raw untyped ptr", there are some usages so we might have to support it.